### ISSUE TITLE
No obvious way to refresh widgets to match python state

### ISSUE THREAD

**Issue by ceball** [ioam/parambokeh#39]

**Comment by philippjfr** [comment]
How does this relate to https://github.com/ioam/param/issues/186?

**Comment by ceball** [comment]
Demonstrated as part of work in https://github.com/ioam/param/pull/250, but I think we can close this issue in favor of [panel](https://github.com/pyviz/panel) :)

**Comment by ceball** [comment]
Actually, I'll leave this open; we should probably deal with parambokeh issues in bulk at some point.

**Comment by philippjfr** [comment]
Agreed, we should transfer any issues that are applicable over to ``pyviz/panel``
